test/fixtures/attributes/index.json

```diff
@@ -22,10 +22,7 @@
               "tagName": "p",
               "properties": {
                 "id": "foo",
-                "className": [
-                  "bar",
-                  "baz"
-                ],
+                "className": ["bar", "baz"],
                 "dataQux": "quux"
               },
               "children": [],
```

Comment:
All looks good, but I am wondering whether we should format JSON: https://github.com/rehypejs/rehype/pull/35#issuecomment-626280946

Reply:
I’d say ignore json too, per the other thread, and then this is good to go!